source/User_Guide/email_activity_beta.md

```diff
@@ -65,34 +133,34 @@ In addition to viewing the email activity associated with your account by recipi
 
 *To use advanced search:*
 
-1. At the top of the Activity Feed page, click **Advanced** search, in the upper right of the search window.
+1. At the top of the Activity Feed page, click **Advanced** search. 
 1. In the field on the far left, select the criteria you wish to filter your event data by. 
-1. Once you have selected all the desired criteria associated with your search, click **Search**. 
-  
-To add additional search criteria, click **Add a Filter**. 
+1. Once you have filled in all of the fields associated with that filter, click **Search**. 
+   To add additional search criteria, click **Add a Filter**. 
 
 {% info %}
 To further narrow your search results, select one of the following options:
 
-* Search emails by **any** of the following - This search is an expansive. It will return more results and will pull in all events that meet *one or more* of the criteria specified by the selected filters.
-* Search emails by **all** of the following - This search is limiting. It will return fewer results and will pull in events that meet *every* criteria specified by the selected filters. 
+* Search emails by **any** of the following - This search will return more results and will pull in all events that meet *one or more* of the criteria specified by the selected filters. 
+or
+* Search emails by **all** of the following - This search will return fewer results and will pull in events that meet *every* criteria specified by the selected filters. 
 {% endinfo %}
 
-{% anchor h2 %}
+{% anchor H2 %}
 Viewing Event History
 {% endanchor %}
 
 *To view event history:*
 
 1. Navigate to the event you wish to view and click on the entry. 
-1. When you have finished reviewing the event data, click **close**. 
+1. When you have finished reviewing the data, click **close**. 
```

Comment:
May want to mention they can click to expand sections for more details? This is where they'd find the clicked URL